src/components/Slider/Slider.vue

```diff
@@ -162,9 +164,37 @@ export default {
 
       this.handleValueChange();
     },
+    checkInputWidth() {
+      const intervalId = setInterval(() => {
+        const inputElement = this.$refs.input;
+        if (inputElement) {
+          const { offsetWidth } = inputElement;
+          if (offsetWidth > 0) {
+            this.sliderWidth = offsetWidth;
+            this.configureTooltip();
+            clearInterval(intervalId);
+          }
+        }
+      }, 100);
+    },
+    checkTooltipLabelWidth() {
+      const intervalId = setInterval(() => {
+        if (this.$refs.tooltip && this.$refs.tooltip.$refs.label) {
+          const { clientWidth } = this.$refs.tooltip.$refs.label;
+          if (clientWidth > 0) {
+            this.labelWidth = clientWidth;
+            this.configureTooltip();
+            clearInterval(intervalId);
+          }
+        }
```

Comment:
This way I imagine it would be more readable and scalable:
```suggestion
        const tooltipLabel = this.$refs.tooltip?.$refs.label
        if (!tooltipLabel) {
          return;
        }
        
        const { clientWidth } = tooltipLabel;
        if (clientWidth > 0) {
          this.labelWidth = clientWidth;
          this.configureTooltip();
          clearInterval(intervalId);
        }
```